client/components/main/layouts.jade

```diff
@@ -13,6 +13,13 @@ template(name="userFormsLayout")
     h1.at-form-landing-logo
       img(src="{{pathFor '/wekan-logo.png'}}" alt="Wekan")
     +Template.dynamic(template=content)
+    div.at-form-lang
+      select.select-lang.js-userform-set-language
```

Comment:
Actually I didn’t try your implementation locally. I think the way you made it is elegant, so let’s keep it like that. We may reconsider based on the rationale above when implementing #51.